### ISSUE TITLE
Fine-tuning with imbalanced classes

### ISSUE THREAD

**Issue by ge96lip** [ge96lip/Explore-Transfer-Learning#6]
For the last exercise check what happens if you have imbalanced classes and try to fine-tune. One option would be to just use 20% of the training images for each cat breed. If you train with the normal cross-entropy loss, what happens to the final test performance on the classes with limited data? You should then try a strategy such as weighted cross-entropy and/or over-sampling of the minority classes to compensate for the imbalanced training set.
